electron/html/certificate-error.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<!-- Wire, Copyright (C) 2016 Wire Swiss GmbH -->
+<html>
+  <head>
+    <style>
+      html {
+        width: 100%;
+        height: 100%;
+      }
+      body {
+        width: 100%;
+        height: 100%;
+        margin: 0;
+        padding: 0;
+        background-image: linear-gradient(0, #2391d3, #68b3e0);
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        font-family: -apple-system, ".SFNSText-Regular", "Helvetica Neue", "Segoe UI", Arial, sans-serif;
+        font-size: 16px;
+        font-weight: 300;
+        text-transform: uppercase;
+        color: #fff;
+      }
+      a, a:hover, a:active, a:visited {
+        color: #fff;
+      }
+    </style>
+  </head>
+  <body ondragstart="return false;" ondrop="return false;">
+    <div id="errormessage">
+      Your connection is not private. <a href="https://wearezeta.zendesk.com/knowledge/articles/115002284109/en-us?brand_id=162184" target="_blank">Learn more</a>
```

Comment:
We can do that later as we didn't have anything before the [about changes](https://github.com/wireapp/wire-desktop/blob/master/electron/js/about.js#L25-L31) last week..